### PR TITLE
feat: added exam timetable service bus configurations

### PR DESCRIPTION
**Pull request by Yakitrak**
## Describe your changes

Ticket: https://pins-ds.atlassian.net/browse/ASB-1847

Added required configurations for nsip-exam-timetable topic on service bus

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] My code follows the style guidelines of this project
- [x] My code changes do not include any hardcoded secrets or passwords
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My Tech Lead is aware of any infrastructure changes that will cost money

### CODE REVIEW

File: app/stacks/uk-west/applications-service/terragrunt.hcl [hunk 1]
```diff
@@ -55,6 +56,7 @@ inputs = {
   back_office_service_bus_register_nsip_subscription_topic_id   = dependency.back_office_ukw.outputs.servicebus_topic_register_nsip_subscription_id
   back_office_service_bus_nsip_project_update_topic_id          = dependency.back_office_ukw.outputs.service_bus_nsip_project_update_topic_id
   back_office_service_bus_nsip_representation_topic_id          = dependency.back_office_ukw.outputs.service_bus_nsip_representation_id
+  back_office_service_bus_nsip_exam_time_table_topic_id         = dependency.back_office_ukw.outputs.service_bus_nsip_exam_time_table_id
```
Comment (JamesP-KC) [inline app/stacks/uk-west/applications-service/terragrunt.hcl:59]: The output from the back-office stack is named `service_bus_nsip_exam_timetable_id`, so will need to be named as such here:
```suggestion
  back_office_service_bus_nsip_exam_time_table_topic_id         = dependency.back_office_ukw.outputs.service_bus_nsip_exam_timetable_id
```

(the output is defined in `app/stacks/uk-west/back-office/outputs.tf` line 129)

File: app/stacks/uk-south/applications-service/terragrunt.hcl [hunk 2]
```diff
@@ -80,6 +81,7 @@ inputs = {
   #  back_office_service_bus_register_nsip_subscription_topic_id    = dependency.back_office_uks.outputs.servicebus_topic_register_nsip_subscription_id
   #  back_office_service_bus_nsip_project_update_topic_id           = dependency.back_office_uks.outputs.service_bus_nsip_project_update_topic_id
   #  back_office_service_bus_nsip_representation_topic_id           = dependency.back_office_uks.outputs.service_bus_nsip_representation_topic_id
+  #  back_office_service_bus_nsip_exam_timetable_topic_id           = dependency.back_office_uks.outputs.service_bus_nsip_exam_timetable_topic_id
```
Comment (JamesP-KC) [inline app/stacks/uk-south/applications-service/terragrunt.hcl:84]:
```suggestion
  #  back_office_service_bus_nsip_exam_timetable_topic_id           = dependency.back_office_uks.outputs.service_bus_nsip_exam_timetable_id
```

File: app/stacks/uk-west/applications-service/terragrunt.hcl (hunk 1 above)
Comment (JamesP-KC) [inline app/stacks/uk-west/applications-service/terragrunt.hcl:59]: naming on the left side here needs to match what is defined in `app/stacks/{region}/applications-service/variables.tf`, as below:

```suggestion
  back_office_service_bus_nsip_exam_timetable_topic_id         = dependency.back_office_ukw.outputs.service_bus_nsip_exam_time_table_id
```

File: app/stacks/uk-south/applications-service/terragrunt.hcl [hunk 3]
```diff
@@ -62,6 +62,7 @@ dependency "applications_service_ukw" {
 #    servicebus_topic_register_nsip_subscription_id                 = "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/mock_resource_group/providers/Microsoft.ServiceBus/namespaces/mock-namespace/topics/mock-topic"
 #    servicebus_topic_deadline_submission_topic_id                  = "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/mock_resource_group/providers/Microsoft.ServiceBus/namespaces/mock-namespace/topics/mock-topic"
 #    service_bus_nsip_project_update_topic_id                       = "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/mock_resource_group/providers/Microsoft.ServiceBus/namespaces/mock-namespace/topics/mock-topic"
+#    service_bus_nsip_exam_timetable_topic_id                       = "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/mock_resource_group/providers/Microsoft.ServiceBus/namespaces/mock-namespace/topics/mock-topic"
```
Comment (JamesP-KC) [inline app/stacks/uk-south/applications-service/terragrunt.hcl:65]:
```suggestion
#    service_bus_nsip_exam_timetable_id                       = "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/mock_resource_group/providers/Microsoft.ServiceBus/namespaces/mock-namespace/topics/mock-topic"
```

**Review by JamesP-KC — COMMENTED** [review]
couple of comments on the naming in terragrunt file, just need to make sure the naming of variables (applications) and outputs (back office) are aligned with the respective `variables.tf` and `outputs.tf` files.

File: app/stacks/uk-west/applications-service/terragrunt.hcl (hunk 1 above)
Comment (JamesP-KC) [reply app/stacks/uk-west/applications-service/terragrunt.hcl]: I think github has marked this as outdated/resolved, but still showing as the old value to me

Comment (Yakitrak) [reply app/stacks/uk-west/applications-service/terragrunt.hcl]: Ah I tried to use the "accept suggestion" button and thought it worked. Good catch! Done manually now.

**Review by JamesP-KC — APPROVED** [review]

**Pull request merged**